h2o-py/tests/testdir_algos/xgboost/pyunit_xgboost_pojo.py

```diff
@@ -25,7 +25,7 @@ def compare_preds(train, test, x, y, booster, ntrees, max_depth, max_error):
     pyunit_utils.compare_frames_local(pred_h2o, pred_pojo, 1, tol=max_error)
 
 
-def test_booster(booster, max_error=1e-10):
+def test_booster(booster, max_error=1e-6):
```

Comment:
is this good enough?

Reply:
Zuzana:  Talked to michalk about this.  The accuracy change is due to change from float to double or vice versa in XGBoost.  It has nothing to do with my code.  He is okay with that.